src/views/movieDetailsView/movieDetailsView.js

```diff
@@ -0,0 +1,211 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import CustomButton from '../../components/common/customButton/customButton'
+import WikiBadge from '../../components/common/customBadges/customBadges'
+
+import { fetchMovieDetails } from '../../actions/movieDetails';
+import { goBack } from '../../actions/navigation';
+
+import { API_PARAMS } from '../../common/constants'
+
+import './movieDetailsView.scss';
+
+
+class MoviesDetailsView extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            showPlot: false
+        };
+    }
+
+    getMovieDetails() {
+        const urlParamId = this.props.match.params.id;
+        const { movieDetails } = this.props.movieDetails;
+
+        if (!movieDetails
+            || (movieDetails.imdbID !== urlParamId)) {
+            const options = {
+                [API_PARAMS.ID]: urlParamId
+            }
+            this.props.fetchMovieDetails(options);
+        }
+    }
+
+    togglePlot = () => {
+        this.setState( (prevState) => {
+            return {
+                showPlot: !prevState.showPlot
+            }
+        });
+    }
+
+    renderPlot = (movieDetails) => {
+        return (
+            <div>
+                <h6><strong>Plot:</strong></h6>
+                <p>
+                    { this.state.showPlot ? movieDetails.Plot : movieDetails.Plot.substr(0, 20) }
```

Comment:
I know ESLint could help here. I'm quite into details and once there is space after `{` the other time there is not `{`. 
Please make it consistent. I'd prefer no space before/after `{`, `}`.